packages/x-components/package.json

```diff
@@ -114,9 +116,11 @@
     "rollup-plugin-vue": "~5.1.9",
     "sass": "~1.49.7",
     "sass-loader": "~8.0.0",
+    "tailwindcss": "~3.1.8",
     "ts-jest": "~27.0.7",
     "ts-node": "~8.9.1",
     "typescript": "~4.6.2",
+    "vue-cli-plugin-tailwind": "~3.0.0",
```

Comment:
With everything now in place do we need this dependency?

Reply:
Actually not, I will remove it.